Kernel/Term.cpp

```diff
@@ -15,15 +15,16 @@
  * @since 06/05/2007 Manchester, changed into a single class instead of three
  */
 
-#include "Lib/Output.hpp"
-#include "Indexing/TermSharing.hpp"
-#include "Lib/Metaiterators.hpp"
-#include "Lib/StringUtils.hpp"
-
 #include "SubstHelper.hpp"
 #include "TermIterators.hpp"
 #include "RobSubstitution.hpp"
+
+#include "Indexing/TermSharing.hpp"
 #include "Kernel/HOL/HOL.hpp"
+#include "Lib/Metaiterators.hpp"
+#include "Lib/Output.hpp"
+#include "Lib/StringUtils.hpp"
+#include "Shell/AnswerLiteralManager.hpp"
```

Comment:
I am not too much in favor of having higher-level structures like `AnswerLiteralManager` in the implementation of something so low-level as `Term`. Can everything that requires `AnswerLiteralManager` be moved to e.g. `AnswerLiteralManager` itself?